### ISSUE TITLE
Sample_Decode: [ERROR], sts=MFX_ERR_UNKNOWN(-1), AllocFrames, m_pGeneralAllocator->Alloc failed

### ISSUE THREAD

**Issue by KalaipriyaRajesh** [Intel-Media-SDK/MediaSDK#2808]
Hi,

I have to check for the pipeline decode->VPP->Display.
So tried Sample_decode code in my linux machine.

This is the error that Iam getting : allocation fails for decoder

mamatha@Rental32-LaptopU:~/msdk/build/__bin/release$ **./sample_decode h264 -i ./input_file.h264 -hw -vaapi -calc_latency -async 2 -rdrm**
libva info: VA-API version 1.13.0
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_7
libva info: va_openDriver() returns 0
drmrender: trying connection: eDP
drmrender: selected crtc already attached to connector
drmrender: succeeded...
drmrender: connected via eDP to 1920x1080@60 capable display
pretending that aspect ratio is 1:1

[ERROR], sts=MFX_ERR_UNKNOWN(-1), AllocImpl, sts!=MFX_ERR_NONE error at /home/mamatha/msdk/samples/sample_common/src/general_allocator.cpp:148

[ERROR], sts=MFX_ERR_UNKNOWN(-1), AllocFrames, m_pGeneralAllocator->Alloc failed at /home/mamatha/msdk/samples/sample_decode/src/pipeline_decode.cpp:1276

[ERROR], sts=MFX_ERR_UNKNOWN(-1), Init, AllocFrames failed at /home/mamatha/msdk/samples/sample_decode/src/pipeline_decode.cpp:595

[ERROR], sts=MFX_ERR_UNKNOWN(-1), main, Pipeline.Init failed at /home/mamatha/msdk/samples/sample_decode/src/sample_decode.cpp:807

whether there is anything to change in options that I have given? 
Kindly help in resolving. Thanks in advance.

**Comment by KalaipriyaRajesh** [comment]
Thanks. Referring that I have changed output format to rgb4 then it worked

**Comment by dmitryermilov** [comment]
Yes, RGB of course shall work. But I thought the point of your issue was to use NV12 specifically because when we having RGB output it means VPP color conversion NV12->RGB (which has some cost).

**Comment by KalaipriyaRajesh** [comment]
We are trying in linux kernel version : 5.4.0-87-generic
